frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -75,6 +76,8 @@ local symbol_prefix = {
         mem_usage = C_("FooterLetterPrefix", "M:"),
         -- @translators This is the footer letter prefix for Wi-Fi status.
         wifi_status = C_("FooterLetterPrefix", "W:"),
+        -- @translators This is the footer letter prefix for the page turn button status.
+        page_turning_inverted = C_("FooterLetterPrefix", "PgT:"),
```

Comment:
Looks like `P:` is available? (and P reads as Page in English and French :))

Reply:
there are two `R:`s, one of those `percentage read` should be `P:`. Honestly I am proud of `PgT`

Reply:
> Honestly I am proud of `PgT`

And you should be :)
I'm just thinking of consistency, all of them are 1 or 2 uppercase letter. `PgT` introduce another style. `PT` would work (even if I would read it as Pass-Through).

Reply:
`PgT` for off and `PgTI` for on? What about the double Rs?

Reply:
> `PgT` for off and `PgT:I` for on?

Feels strange. Do we already have this behaviour?

Also, hear your first customer :) _could it be something more compact than previously shown in screenshots?_

> What about the double Rs?

Should probably be fixed, and one must change. (I would change frontlight_warmth - to what I don't know - because it's the one probably the less used/seen, so will hurt less people. "R" is fine for Reading percentage, "P" because of Percentage it ambiguous, battery has a percentage too).

Reply:
"R" is for Redshift, said @zwim.

Reply:
Only the science guy @zwim is will know what Redshift means :)
(I thought it was an available letter picked from "waRmth" :))

Reply:
okay how about this, (because I really, really like it) `PgT` and `PgT:I` as icons (I know, I know), the arrows as compact, and then letters could be `PT` or whatever...

also, I am the first customer, patient zero...

Reply:
I'll let others share their thoughts.
(I value consistency, and not having various looks - if you really have your way with PgT, use it in all modes.)